### PR TITLE
[SSH Connection Manager] Add custom SSH port

### PR DESCRIPTION
**Pull request by matt08**
## Description

Add option to use custom SSH port (other than `22`) in SSH connections in SSH Connection Manager extension.

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @matt08 👋 

Thanks for your contribution 🔥 

@scopsy could you give it a spin and review it?

File: extensions/ssh-manager/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # ssh-manager Changelog
 
+## [Add support to custom port] - 2022-07-07
+This verson adds option to set custom port (other than `22`) in SSH connections.
```
Comment (scopsy) [inline extensions/ssh-manager/CHANGELOG.md:4]:
```suggestion
This version adds option to set custom port (other than `22`) in SSH connections.
```

**Review by scopsy — APPROVED** [review]
Looks great! Only a small typo

**Review by pernielsentikaer — APPROVED** [review]
Hi @matt08 👋 

Approved 🔥

**Pull request merged**